packages/rainbowkit/src/wallets/walletConnectors/mewWallet/mewWallet.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { InjectedConnector } from 'wagmi/connectors/injected';
+import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
+import { isAndroid } from '../../../utils/isMobile';
+import { Wallet } from '../../Wallet';
+import { getWalletConnectConnector } from '../../getWalletConnectConnector';
+export interface MewWalletOptions {
+  chains: Chain[];
+  shimDisconnect?: boolean;
+}
+
+export const mewWallet = ({
+  chains,
+  shimDisconnect,
+}: MewWalletOptions): Wallet => ({
+  id: 'mewWallet',
+  name: 'MEW wallet',
```

Comment:
Can we replace this with `MEW Wallet` for consistency, or is this the correct brand representation?